pkg/kubernetes/client_test.go

```diff
@@ -352,7 +352,179 @@ var _ = Describe("Test Namespace KubeController Methods", func() {
 			Expect(svcAccounts).Should(HaveLen(len(expectedSvcAccounts)))
 			Expect(svcAccounts).Should(ConsistOf(expectedSvcAccounts))
 		})
-
 	})
 
+	Context("Test ListPodsForService()", func() {
+		var kubeClient *testclient.Clientset
+		var kubeController Controller
+		var err error
+		testMeshName := "foo"
+
+		BeforeEach(func() {
+			kubeClient = testclient.NewSimpleClientset()
+			kubeController, err = NewKubernetesController(kubeClient, testMeshName, make(chan struct{}))
+			Expect(err).ToNot(HaveOccurred())
+			Expect(kubeController).ToNot(BeNil())
+		})
+
+		It("should correctly return the pods associated with a service", func() {
```

Comment:
What's the difference between this test and the test below? Can you include that in the text here? Or in a comment

Reply:
I left an additional comment.